docs/source/api/deliveryservices.rst

```diff
@@ -51,7 +51,11 @@ Response Structure
 ------------------
 :active:                   ``true`` if the Delivery Service is active, ``false`` otherwise
 :anonymousBlockingEnabled: ``true`` if :ref:`Anonymous Blocking <anonymous_blocking-qht>` has been configured for the Delivery Service, ``false`` otherwise
-:cacheurl:                 Deprecated. A setting for a deprecated feature of now-unsupported Trafficserver versions
+:cacheurl:                 A setting for a deprecated feature of now-unsupported Trafficserver versions
+
+	.. deprecated:: ATCv3.0
+	This field has been deprecated in Traffic Control 3.x and is subject to removal in Traffic Control 4.x or later
```

Comment:
So I think this may be my fault. The "normal" syntax for the `.. deprecated` directive is 

```rst
.. deprecated:: version A description of the deprecation.
```
... but because it treats the first string (whitespace-delimited) specially, lots of people (myself included) prefer to write it as:

```rst
.. deprecated:: version
	A description of the deprecation.
```

However, that may somewhat obscure the fact that the description must fall within the directive block. Essentially what I'm saying is that you need to indent that second line one more time. That will cause it to be treated as a part of the `.. deprecated` directive block.